packages/core/ios/Sources/DdSdkImplementation.swift

```diff
@@ -229,6 +231,13 @@ public class DdSdkImplementation: NSObject {
             )
         }
         
+        var customRUMEndpointURL: URL? = nil
+        if let customRUMEndpoint = configuration.customEndpoints?.rum as? NSString {
+            if (customRUMEndpoint != "") {
+                customRUMEndpointURL = URL(string: "\(customRUMEndpoint)/api/v2/rum" as String)
```

Comment:
On the android side we have [something like this](https://github.com/DataDog/dd-sdk-android/blob/develop/features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/net/RumRequestFactory.kt#L67)

Reply:
Yes, there is a discrepancy here between iOS (does not add this path) and Android (always adds this path). 
I prefer to have a consistent behaviour across both platforms, this is what we do on Flutter as well.
Otherwise it's up to the user to add this path for iOS only.

Reply:
Hello there. Does that mean our endpoint *must* confirm to this `domain.tld/api/v2/rum` path?
Thank you 🙂

Reply:
Hi! 
Yes indeed.

This is the best compromise we could find to deliver this feature to customers that are willing to try it in a reasonable time.

I'll detail that more in the related issues :)

Reply:
Thanks for your answer, @louiszawadzki!